### ISSUE TITLE
MQTT broker client id not customizable in hardware configuration

### ISSUE THREAD

**Issue by whren** [stas-demydiuk/domoticz-zigbee2mqtt-plugin#44]
Hi,

I have  a MQTT broker with client id prefix filtering setup and I've not seen any possibility to defined it in domoticz hardware configuration.
I tracked the [source](https://github.com/stas-demydiuk/domoticz-zigbee2mqtt-plugin/blob/master/mqtt.py#L50) and think ID value should be configuration based before relying on the actual behavior as a fallback option.

**Comment by stas-demydiuk** [comment]
Should be available with plugin in latest master branch, could you please update and check

**Comment by whren** [comment]
Yes, confirmed working on latest master branch, thank you configuration aligned with zigbee2mqtt now !
